disnake/welcome_screen.py

```diff
@@ -74,17 +74,10 @@ def _from_data(
         *,
         data: WelcomeScreenChannelPayload,
         state: ConnectionState,
-        guild: Union[Guild, PartialInviteGuild],
     ) -> WelcomeScreenChannel:
         emoji_id = utils._get_as_snowflake(data, "emoji_id")
-        emoji_name = data.get("emoji_name") or None
-        emoji = None
-        if emoji_name:
-            emojis: Optional[Tuple[Emoji]]
-            if emojis := getattr(guild, "emojis", None):
-                emoji = utils.get(emojis, id=emoji_id, name=emoji_name)
-            if not emoji:
-                emoji = PartialEmoji.with_state(state, name=emoji_name, id=emoji_id)
+        emoji_name = data.get("emoji_name")
+        emoji = state._get_emoji_from_fields(name=emoji_name, id=emoji_id)
```

Comment:
style inconsistency between this and the above

Reply:
https://github.com/DisnakeDev/disnake/pull/980/commits/3819f3eed4cc25fc8a979109e02d933263291d4f